### PR TITLE
Change Keras serialization KERAS_H5 to KERAS

### PR DESCRIPTION
**Pull request by reuvenperetz**
## Pull Request Description:

Change Keras serialization KERAS_H5 to KERAS, and take code out from exporter README to new notebooks demonstrating the exporter usage.

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/notebooks/example_keras_export.ipynb [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "\n",
+        "#### Fakely-Quantized TFLite\n",
+        "\n",
+        "The model will be exported as a tflite model where weights and activations are quantized but represented as float.\n",
+        "operators.\n",
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_export.ipynb:209]: operators? float operators?

File: tutorials/notebooks/example_keras_export.ipynb [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+        "id": "voOrtCroD-HE"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bb7e1572"
+      },
+      "source": [
+        "Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.\n",
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_export.ipynb:256]: 2024?

File: tutorials/notebooks/example_pytorch_export.ipynb [hunk 3]
```diff
@@ -0,0 +1,231 @@
+{
```
Comment (elad-c) [inline tutorials/notebooks/example_pytorch_export.ipynb:1]: Same as keras

**Review by elad-c — APPROVED** [review]

File: tutorials/notebooks/example_keras_export.ipynb (hunk 1 above)
Comment (reuvenperetz) [reply tutorials/notebooks/example_keras_export.ipynb:209]: I'll change it to data types. It will be clearer

**Pull request merged**